ambry-server/src/test/java/com.github.ambry.server/StatsManagerTest.java

```diff
@@ -219,23 +238,30 @@ public void testShutdownBeforeStart() throws InterruptedException {
 
   /**
    * Generate a random, two levels of nesting (accountId, containerId) {@link StatsSnapshot} for testing aggregation
-   * @return a {@link StatsSnapshot} with random structure and values
+   * @return a map of all types of {@link StatsSnapshot} whose key is the type name and value is corresponding snapshot
    */
-  private StatsSnapshot generateRandomSnapshot() {
+  private Map<StatsReportType, StatsSnapshot> generateRandomSnapshot() {
     Map<String, StatsSnapshot> accountMap = new HashMap<>();
+    Map<String, StatsSnapshot> account_container_Map = new HashMap<>();
```

Comment:
minor: naming with camel case, not underscore

Reply:
will fix.